praw/models/base.py

```diff
@@ -42,3 +42,17 @@ def __init__(self, reddit: Reddit, _data: Optional[Dict[str, Any]]):
         if _data:
             for attribute, value in _data.items():
                 setattr(self, attribute, value)
+
+    def __eq__(self, other):
+        """Check if a PRAWBase is equal to another PRAWBase."""
```

Comment:
I'm not sure if it makes sense to do this across all instances of RedditBase since Items can be lazily loaded. It wouldn't make sense to compare two equal submissions when lazily loaded, and then load them and have them not equal. As a result, I think any instances that want to be compared should have an explicit `__eq__` provided.

Reply:
This makes sense.